src/studio/src/designer/DataModeling.Tests/_TestData/Model/JsonSchema/General/ComplexSchema-old.json

```diff
@@ -104,10 +104,10 @@
     "tags": {
       "type": "array",
       "items": {
-        "type": "string"
-      },
-      "@xsdType": "string",
-      "@xsdStructure": "0.Schema/4.SimpleType"
+        "type": "string",
```

Comment:
I'd suggest deleting "ComplexSchema-old.json" file rather than making changes to it. It's not used in backend tests. More so, that file is not related to new converters.

Reply:
Ok, removing changes to this file. Not deleting it, think that is for the converter side to deceide.